assets/chat/js/messages/ChatEventMessage.js

```diff
@@ -16,8 +16,8 @@ export default class ChatEventMessage extends ChatMessage {
       .querySelector('#event-template')
       ?.content.cloneNode(true).firstElementChild;
 
-    if (this.user && this.user.username)
-      eventTemplate.dataset.username = this.user.username.toLowerCase();
+    if (this.user && this.user.displayName)
+      eventTemplate.dataset.username = this.user.displayName;
```

Comment:
Does this fix an issue where user messages in subscription events aren't uncensored?